Detectors/MUON/MID/Base/include/MIDBase/GeometryTransformer.h

```diff
@@ -28,13 +29,9 @@ namespace mid
 class GeometryTransformer
 {
  public:
-  GeometryTransformer();
-  virtual ~GeometryTransformer() = default;
-
-  GeometryTransformer(const GeometryTransformer&) = delete;
-  GeometryTransformer& operator=(const GeometryTransformer&) = delete;
-  GeometryTransformer(GeometryTransformer&&) = delete;
-  GeometryTransformer& operator=(GeometryTransformer&&) = delete;
+  void setMatrix(int deId, const ROOT::Math::Transform3D& matrix);
```

Comment:
No showstoppers, but something to think about maybe. 

1. If when constructing the transformations your inputs will always be chamber transformations * RPC transformations, why not having a setMatrix(int de, chtrans, rpctrans) instead ?

2. That setMatrix makes no check whatsoever to the deId given to it... A doc comment with the accepted range of deId would be the least to do...

Reply:
Hi @aphecetche ,
concerning 1., I prefer to keep it like this. The chtrans, rpctrans convention is needed when building the geometry in gGeoManager, since it reflects the way the volumes are built. However, the GeometryTransformer is more general, and one can think about directly creating a RPC matrix instead. So I prefer to keep the current constructor.
Concerning 2, I'll add a protection